github-event-service/index.js

```diff
@@ -0,0 +1,4 @@
+"use strict"
+// This shim file is in place to allow for deploying properly.
+Object.defineProperty(exports, "__esModule", { value: true });
```

Comment:
Why does this need the `__esModule` property? I'm pretty sure it runs without as well?

Reply:
You're right, it doesn't.  Honestly, I just copy pasted that portion from the generated js files from a tsc build.